web/crux-ui/src/elements/dyo-modal.tsx

```diff
@@ -34,7 +34,7 @@ const DyoModal = (props: DyoModalProps) => {
       >
         <Dialog.Overlay />
 
-        <DyoCard className={clsx(props.className, 'flex flex-col m-auto p-8')}>
+        <DyoCard className={clsx(props.className, 'flex flex-col m-auto p-8')} modal={true}>
```

Comment:
`modal` is enough

Reply:
The reason why I am not an FE dev, thanks the point out!